### ISSUE TITLE
The client-side enable method does not enable a disabled ContextMenu item

### ISSUE THREAD

**Issue by aleksandarevangelatov** [telerik/kendo-ui-core#6899]
### Bug report

When using the Telerik UI for ASP.NET Core ContextMenu wrapper and an item is initialized as disabled, calling the client side enable method does not enable the item.

### Reproduction of the problem
1. Initialize a disabled item:
```
items.Add()
  .Text("Reply")
  .ImageUrl(Url.Content("~/shared/web/toolbar/reply.png"))
  .HtmlAttributes(new { id = "ReplyID", title = "Reply" })
  .Enabled(false)
  .Items(children =>
  {
      children.Add().Text("Reply To Sender").ImageUrl(Url.Content("~/shared/web/toolbar/reply.png"));
      children.Add().Text("Reply To All").ImageUrl(Url.Content("~/shared/web/toolbar/reply.png"));
  });
```
2. Call the [enable](https://docs.telerik.com/kendo-ui/api/javascript/ui/contextmenu/methods/enable) method:
```
var menu = $("#menu").data("kendoContextMenu");
menu.enable("#ReplyID", true);
```
[Example](https://netcorerepl.telerik.com/GckKwNYL27SD63qf36)
### Current behavior
The ContextMenu item is not enabled

### Expected/desired behavior
The ContextMenu item is enabled

### Workaround
Remove the `k-state-disabled` class from the element with id `ReplyID`
```
$("#ReplyID").removeClass("k-state-disabled")
```

### Environment

* **Kendo UI version:** 2022.2.621
* **Browser:** [all]

**Comment by aleksandarevangelatov** [comment]
Will be fixed with https://github.com/telerik/kendo/pull/15381

**Comment by Dimitar-Goshev** [comment]
The issue is fixed with 2022.3.913
